### ISSUE TITLE
MemberOrderHelper.GetAccessLevelForOrdering does not consider explicit event implementations

### ISSUE THREAD

**Issue by vweijsters** [DotNetAnalyzers/StyleCopAnalyzers#1823]
Found during review of #1812.

Although explicitly implemented events are rare (since you can't use the event field declaration syntax), I think they should be treated the same as all other explicitly implemented members.

**Comment by sharwell** [comment]
I agree with treating them the same. Let's start my checking what accessibility StyleCop Classic treats them as, so we know whether or not this would be a deviation.

**Comment by Noryoko** [comment]
Classic treats them as private. See #1179 and #1180.

**Comment by sharwell** [comment]
I updated this issue to be treated as a proposal, to which I vote :+1:
